proto/kvrpcpb.proto

```diff
@@ -862,6 +862,10 @@ message Context {
     // The keyspace that the request is sent to.
     // NOTE: This field is only meaningful while the api_version is V2.
     uint32 keyspace_id = 32;
+    
+    // It tells us where the request comes from in TiDB. If it isn't from TiDB, leave it blank.
+    // This is for tests only and thus can be safely changed/removed without affecting compatibility.
+    SourceStmt source_stmt = 33;
```

Comment:
If this field is only used for logging with context info, I think a formated string is enough (just like `request_source`).

Reply:
IMO string is less structured and less flexible.